src/VisualStudio/CSharp/Test/ProjectSystemShim/CPS/AdditionalPropertiesTests.cs

```diff
@@ -29,5 +32,49 @@ public void SetProperty_RootNamespace_CPS()
             static string DefaultNamespaceOfSingleProject(TestEnvironment environment)
                 => environment.Workspace.CurrentSolution.Projects.Single().DefaultNamespace;
         }
+
+        [WpfFact]
+        [Trait(Traits.Feature, Traits.Features.ProjectSystemShims)]
+        public void SetProperty_MaxSupportedLangVersion_CPS()
+        {
+            var catalog = TestEnvironment.s_exportCatalog.Value
+                .WithParts(
+                    typeof(CSharpParseOptionsChangingService));
```

Comment:
Why can't we just add this to the main TestEnvironment catalog? I think there's a "with derived types" helper we use for including language services like this.

Reply:
It requires a project reference to `Microsoft.VisualStudio.LanguageService.CSharp` 

That's to say, I'm not opposed just lazy.